posts/2022-12-06-automation-star-conference.adoc

```diff
@@ -0,0 +1,132 @@
+
+== Conference Program
+
+The https://automation.eurostarsoftwaretesting.com/conference/2022/programme/[conference program] included two days of Tutorials, keynotes, and talks led by experts from around the testing world. It also included exclusive networking and social events like lean coffee and Community Conversations. Prior to the conference, attendees have the option to choose the Tutorials and the sessions in the Whova App and from the app they can also download the slides for the presentations. The tutorials and talks were in German and English so attendees can select according to their preferred language. The tutorials were scheduled for half day and the talks were for forty-five minutes.
+
+== Begins the Conference
+
+The first day of the conference started with the Badge and goodies collection and morning coffee. Then attendees were directed to their selected tutorials which were scheduled for half day. Second half of the first day and the second day of the conference was the combination of Keynotes and talks. All attendees get to choose between different talks happening at the same time in separate conference rooms. At a time participants can attend one out of five talks.
+
+== Talks at Conference:
+
+In the following, I would like to briefly discuss the talks that we have attended:
+
+=== 1. Making Great Waves: 8 Software Testing Convictions
+
+Testing is an art as much as Engineering, we can take cues from prolific artists in how we approach quality in our own work. In this keynote *Andrew Knight* mentioned how we can create our own great waves in software testing. He covered following eight auspicious ideas which were inspired by ukiyo-e printing:
+
+. Focus on behavior
+. Prioritize on risk
+. Automate
+. Shift left and right
+. Give fast feedback
+. Go lean
+. Open up
+. Show empathy
+
+=== 2. WebDriver BiDi – The Future of Cross-Browser Automation
+
+Pain of browser automation was explained in a very coolest way by *Jecelyn Yeen*. She started with the history of web and automation, followed by how browser automation works behind the scenes and then explained the pros and cons of different ways of browser automation. In the end she listed her experiences with the WebDriver BiDi project and its current progress.
+
+=== 3. Mission Impossible: Automatic Usability Testing
+
+If usability testing is part of the project then there are more chances of developing successful product. *Kimberly Snoyl* has delivered the talk focusing on the importance of UX testing, because sometimes the products are not user-friendly and users get frustrated easily while using it. To make processes simpler and better for the users, Kimberly is exploring the idea to create a tool which can automate some of the following 10 Usability Heuristics:
+
+image::../media/2022-12-06-automation-star-conference/how to test UX.png[alt="Figure 1: Usability Heuristics"]
```

Comment:
You don't need to specify the full path, only the image name as you already defined imagesdir on the top ;)